blueprints/user.py

```diff
@@ -0,0 +1,60 @@
+from flask import Blueprint, request, jsonify
+from db import orm
+from models import User
+
+user_bp = Blueprint('user_bp', __name__)
```

Comment:
```suggestion
user_bp = Blueprint('user_bp', __name__, url_prefix='/users')
```

You can use `url_prefix` parameter to Blueprint class. It will add **"/users"** to all other endpoints which were created using @blog_bp automatically. Now you can remove the **"/users"** from each route.

Endpoints then look like:

@user_bp.route('/', methods=['GET'])
@user_bp.route('/', methods=['POST'])
@user_bp.route('/<int:user_id>', methods=['PUT'])
@user_bp.route('/<int:user_id>', methods=['DELETE'])